normandy/base/tests/api/test_api.py

```diff
@@ -235,3 +236,121 @@ def test_logged_out(self, settings):
             "logout_url": settings.OIDC_LOGOUT_URL,
             "github_url": settings.GITHUB_URL,
         }
+
+
+@pytest.mark.django_db
+class TestUserAPI(object):
```

Comment:
I think that the model viewset also allows creating and deleting users. Can you add tests for that?